### ISSUE TITLE
Can't receive artnet values

### ISSUE THREAD

**Issue by yasdfgr** [Theoriz/art-net-light#1]
I tryed the app on windows and android. I can't receive any artnet values.
All values remain at 0.
What artnet-subnet/universe is used?
Can you provide a step by step guide, please?

**Comment by alexbourgeois** [comment]
Hi, thank you for your feedback!
Currently we are not using any universe/subnet only channels (it is in the roadmap).
I am gonna guide you :
Firstly make sure both the art-net sender and receiver are on the same network and have an IP address.
For the example I will use AuroraDMX on android to send data and the Windows version of Art-Net-Light to receive data.
1- Open the Windows application.
2- Allow the application through your firewall .
3- Open AuroraDMX.
4- In the settings menu set the ArtNet server IP address to the computer one in this format : "192.168.X.X:6454". You can see your IP address in Art-Net light UI. If you don't see the UI just click on the screen or press the "h" key.
5- Play with the 3 first sliders in AuroraDMX to see Art-Net-Light background changing color.

Does it work for you ? It works for me if I follow those steps.

**Comment by yasdfgr** [comment]
Thank you very much.
I tried it again. I used "The ArtNetominator" to diagnose the windows side. I' ve got some values from Aurora on strange net/universe combination. But at first Art-Net-Light doesn't work. Aurora also has a bug on the server-config: my network ist 192.168.26.xxx. It's not possible to enter that address (field to short).
I disabled the windows firewall completly (even the part "public network") and started  Art-Net-Light again.
At the Andoid side i use Art-Net Controller (very nice tool). I played around with broadcasting 255.255.255.255 vs. 192.168.26.255 vs. unicast 192.168.26.99.
After changing the address on Art-Net Controller, Art-Net-Light doesn't react anymore until restart; but only the windows version. On a second device (android app) it worked perfectly.
